src/framebuffer/core.rs

```diff
@@ -50,7 +50,19 @@ impl Framebuffer {
         let device = &*device::CURRENT_DEVICE;
         match device.model {
             Model::Gen1 => Framebuffer::device(device.get_framebuffer_path()),
-            Model::Gen2 => Framebuffer::rm2fb(device.get_framebuffer_path()),
+            Model::Gen2 => {
+                // Auto-select old method still if env LIBREMARKABLE_FB_DISFAVOR_INTERNAL_RM2FB is set affirmatively
+                if let Ok(env_answer) = std::env::var("LIBREMARKABLE_FB_DISFAVOR_INTERNAL_RM2FB") {
+                    let env_answer = env_answer.to_lowercase();
+                    if env_answer == "1" || env_answer == "true" || env_answer == "yes" {
+                        Framebuffer::device(device.get_framebuffer_path())
+                    } else {
+                        Framebuffer::rm2fb(device.get_framebuffer_path())
+                    }
+                } else {
+                    Framebuffer::rm2fb(device.get_framebuffer_path())
+                }
```

Comment:
How about this? (untested)
```suggestion
                match std::env::var("LIBREMARKABLE_FB_DISFAVOR_INTERNAL_RM2FB").ok() {
                    Some("1") => Framebuffer::device(device.get_framebuffer_path()),
                    _ => Framebuffer::rm2fb(device.get_framebuffer_path()),
                }
```

Reply:
(I'm more inclined to be restrictive in the possible values, but feel free to revert that)